### ISSUE TITLE
Propose a dedicated temporal coverage step similar to geographical & taxonomic, from files attributes

### ISSUE THREAD

**Issue by yvanlebras** [earnaud/MetaShARK-v2#67]
Really not important, but maybe to consider, adding a step similar to geographical and taxonomic coverage for temporal coverage so we can ask MetaShARK create this coverage from data file attribute(s)

**Comment by earnaud** [comment]
The fact is that I rely on EAL's way to do. I shall see with @clnsmth about this. However, I think temporal coverage could be improved, as it only covers the period between which the project has been initiated until the data publication date (if I understood it correctly).

**Comment by yvanlebras** [comment]
Yes, I understand the link with emlAL. So it appears to me of interest to see with Colin what he thinks is relevant. Temporal coverage is here describing the entire project time I think, and we can propose a way to describe temporal coverage for each data file and/or on the abstract/methods/additional info 
sections for each distincts project steps

**Comment by clnsmth** [comment]
@yvanlebras @earnaud I'm happy to implement support for temporal coverage of data objects in EMLassemblyline. Would you mind opening an issue in the [project GitHub](https://github.com/EDIorg/EMLassemblyline/issues) describing this feature?

**Comment by yvanlebras** [comment]
Done https://github.com/EDIorg/EMLassemblyline/issues/66
